modules/aerodyn/CMakeLists.txt

```diff
@@ -111,7 +115,7 @@ set(UA_DRIVER_SOURCES
 add_executable(unsteadyaero_driver ${UA_DRIVER_SOURCES})
 target_link_libraries(unsteadyaero_driver aerodynlib fvwlib uaaerolib afinfolib nwtclibs versioninfolib ${CMAKE_DL_LIBS})
 
-install(TARGETS unsteadyaero_driver aerodyn_driver aerodynlib fvwlib uaaerolib afinfolib aeroacoustics
+install(TARGETS unsteadyaero_driver aerodyn_driver aerodynlib fvwlib uaaerolib afinfolib aeroacoustics aerodynlib_obj
```

Comment:
Do we need to install the new objects? As I understand it, they're only used at compile-time to create the new `openfast_prelib_obj`.

Reply:
Yes, intuitively they have to be in the install target to be used in other CMAKE libraries, but they are not actually installed like real libraries are.